pkg/system/system.go

```diff
@@ -577,7 +579,12 @@ func RunStatus(cmd *cobra.Command, args []string) {
 	util.KubeCheck(secret)
 
 	fmt.Println("")
-	fmt.Printf("%s password : %s\n", secret.StringData["email"], secret.StringData["password"])
+	if options.ShowSecrets {
+		fmt.Printf("%s password : %s\n", secret.StringData["email"], secret.StringData["password"])
+	} else {
+		fmt.Printf("%s password : %s\n", secret.StringData["email"],
+			re.ReplaceAllString(secret.StringData["password"], "*"))
```

Comment:
as @liranmauda commented, you can use [MaskedString](https://github.com/noobaa/noobaa-operator/blob/f325c18cfa94b33d54c8e8b05a5edf7ad962615a/pkg/nb/types.go#L18-L22)

```suggestion
		fmt.Printf("%s password : %s\n", secret.StringData["email"], nb.MaskedString(secret.StringData["password"]))
```